src/features/filter.ts

```diff
@@ -1,10 +1,44 @@
-export const actions = { /* put action creators here */};
+import { Status } from '../types/Status';
 
-const filterReducer = () => {
-  return {
-    query: '',
-    status: 'all',
-  };
+enum Type {
+  query = 'add/query',
+  clearQuery = 'add/clear',
+  status = 'add/status',
+}
+
+type QueryAction = { type: Type.query, payload: string };
+type StatusAction = { type: Type.status, payload: string };
```

Comment:
```suggestion
type StatusAction = { type: Type.status, payload: Status };
```
instead of type casting below: 
![image](https://github.com/mate-academy/react_redux-list-of-todos/assets/86923089/7e50a898-e733-4670-ab4c-b530756718fd)